app/src/main/java/com/arygm/quickfix/ui/uiMode/appContentUI/userModeUI/profile/UserProfileScreen.kt

```diff
@@ -0,0 +1,106 @@
+
+@Composable
+fun UserProfileScreen(
+    userNavigationActions: NavigationActions,
+    navigationActions: NavigationActions,
+    rootMainNavigationActions: NavigationActions,
+    preferencesViewModel: PreferencesViewModel,
+    userPreferencesViewModel: PreferencesViewModelUserProfile,
+    appContentNavigationActions: NavigationActions,
+    modeViewModel: ModeViewModel
+) {
+  val isWorker by preferencesViewModel.isWorkerFlow.collectAsState(initial = false)
+
+  // Define Personal Settings
+  val personalSettings =
+      listOf(
+          SettingItemData(
+              icon = Icons.Outlined.Person,
+              label = "My Account",
+              testTag = "AccountConfigurationOption",
+              action = { navigationActions.navigateTo(UserScreen.ACCOUNT_CONFIGURATION) }),
+          SettingItemData(
+              icon = Icons.Outlined.Settings,
+              label = "Preferences",
+              testTag = "Preferences",
+              action = { /* Action */}),
+          SettingItemData(
+              icon = Icons.Outlined.FavoriteBorder,
+              label = "Saved Lists",
+              testTag = "SavedLists",
+              action = { /* Action */}))
+
+  // Define Resources Section
+  val resources =
+      listOf(
+          SettingItemData(
+              icon = Icons.AutoMirrored.Outlined.HelpOutline,
+              label = "Support",
+              testTag = "Support",
+              action = { /* Action */}),
+          SettingItemData(
+              icon = Icons.Outlined.Info,
+              label = "Legal",
+              testTag = "Legal",
+              action = { /* Action */}))
+
+  // Conditional "Become a Worker" Section
+  val conditionalWorkerSection =
+      SettingItemData(
+          icon = Icons.Outlined.WorkOutline,
+          label = "Become a Worker",
+          testTag = "SetupYourBusinessAccountOption",
+          action = { navigationActions.navigateTo(UserScreen.TO_WORKER) })
+
+  // Pass sections as lambdas to `QuickFixProfileScreenElement`
+  QuickFixProfileScreenElement(
+      modeNavigationActions = userNavigationActions,
+      navigationActions = navigationActions,
+      rootMainNavigationActions = rootMainNavigationActions,
+      preferencesViewModel = preferencesViewModel,
+      userPreferencesViewModel = userPreferencesViewModel,
+      appContentNavigationActions = appContentNavigationActions,
+      modeViewModel = modeViewModel,
+      false,
+      AppMode.WORKER,
+      sections =
+          listOf(
+              { modifier ->
+                SettingsSection(
+                    title = "Personal Settings",
+                    items = personalSettings,
+                    screenWidth = 360.dp,
+                    cardCornerRadius = 16.dp)
+              },
+              { modifier ->
+                SettingsSection(
+                    title = "Resources",
+                    items = resources,
+                    screenWidth = 360.dp,
```

Comment:
Watch out here for absolute values ..., you could wrap UserProfileScreen in a `BoxWithConstrains` and use `maxWidth` and `maxHeight`